ansible_base/authentication/utils/claims.py

```diff
@@ -61,7 +61,7 @@ def create_claims(authenticator: Authenticator, username: str, attrs: dict, grou
     logger.debug(f"Authenticator ID: {authenticator.id}")
     maps = AuthenticatorMap.objects.filter(authenticator=authenticator.id).order_by("order")
     logger.debug("==============================================================")
```

Comment:
Can we pop this line out, it never made much sense since there can be multiple auths going on at once.